### ISSUE TITLE
ws4sqlite does not listen on IPv6 socket even when started with an IPv6 address

### ISSUE THREAD

**Issue by Chaz6** [proofrock/ws4sqlite#1]
```
$ ws4sqlite --bind-host '[::]' --mem-db mem1 &
ws4sqlite 0.11.1
- Serving database 'mem1' from :memory:?_journal=WAL
  + No config file loaded, using defaults
  + Using WAL
- Web Service listening on [::]:12321

$ curl https://[::1]:12321
curl: (7) Failed to connect to ::1 port 12321: Connection refused

$ netstat -l -n -p | grep 12321
tcp        0      0 0.0.0.0:12321           0.0.0.0:*               LISTEN      4060941/ws4sqlite
```

**Comment by proofrock** [comment]
Thank you for reporting the issue. I will look into it during the weekend; it must be some misconfiguration of the underlying Fiber stack.

**Comment by proofrock** [comment]
In the meanwhile, please note that https is not directly supported (it's in your curl connection string), see [here](https://germ.gitbook.io/ws4sqlite/security#use-a-reverse-proxy-if-going-on-the-internet).

**Comment by proofrock** [comment]
@Chaz6 I think I found the issue. If I publish the fix in the `develop` branch, are you able to compile it and test? If not, you could tell me which OS/arch your system is and I can provide a binary. Of course, if you'll be so kind to beta-test it; if not I'll try some more and publish it in due time.

Also, connecting to `https://...` hangs curl, so be sure to use `http://`.

**Comment by proofrock** [comment]
Closed in 0.11.2

**Comment by Chaz6** [comment]
That works, thanks!

```
$ netstat -l -n -p | grep 12321
tcp6       0      0 :::12321                :::*                    LISTEN      225268/ws4sqlite
```
